### DISCUSSION TITLE
Block gas limit

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#1037] [General]
### Discussed in https://github.com/0xPolygon/polygon-edge/discussions/1031

<div type='discussions-op-text'>

<sup>Originally posted by **Nexsuperne101** December  9, 2022</sup>
Good evening. Is 1,000,000,000 too high for a block gas limit? The default is 524k. Others have set it at 30 million. The example on the wiki for Edge is 1 billion.

What is the issue if it is too high? Why use 1 billion as an example if it is very bad?

Thanks</div>

**Comment by ghost** [comment]
@laviniat1996 Could I have your expert opinion please?

Thanks, Nex

**Comment by ivanbozic21** [comment]
Hi @Nexsuperne101 

> Good evening. Is 1,000,000,000 too high for a block gas limit? The default is 524k. Others have set it at 30 million. The example on the wiki for Edge is 1 billion.

This really depends on what kind of transaction you plan to perform on your chain. 

Let us know if you will have any more queries.
